pkg/config/config_factory_test.go

```diff
@@ -35,6 +35,10 @@ func TestGetCfgFactoryConfig(t *testing.T) {
 		Factories: factories,
 	}
 
+	resetFlag := func() {
+		configFlag = new(stringArrayValue)
```

Comment:
```suggestion
		configFlag = new(stringArrayValue)
		setFlag    = new(stringArrayValue)
```

Not sure it's used in the tests, but probably best to reset both of these here to avoid confusion if someone adds a test for the set flag later.